cmp/filter/FilterChooserModel.js

```diff
@@ -106,6 +111,7 @@ export class FilterChooserModel extends HoistModel {
         valueSource = bind,
         initialValue = null,
         initialFavorites = [],
+        suggestFieldsWhenEmpty,
```

Comment:
I think worth it to specify '= false' in situations like this for documentation/clarity